dockstore-webservice/src/main/resources/migrations.1.8.0.relinquish.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<!--
+  ~    Copyright 2019 OICR
+  ~
+  ~    Licensed under the Apache License, Version 2.0 (the "License");
+  ~    you may not use this file except in compliance with the License.
+  ~    You may obtain a copy of the License at
+  ~
+  ~        http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~    Unless required by applicable law or agreed to in writing, software
+  ~    distributed under the License is distributed on an "AS IS" BASIS,
+  ~    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~    See the License for the specific language governing permissions and
+  ~    limitations under the License.
+  -->
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd"
+                   context="1.8.0.relinquish">
+    <changeSet id="natalieperez" author="relinquish">
+        <sql dbms="postgresql">
```

Comment:
I still have my doubts about this. Wouldn't it be better to change init_migrations.sh to instead execute as the dockstore user starting with 1.8.migrations? Otherwise what we are doing is:

1. Start with copy of production database with dockstore as owner of everything
2. Run 1.8 migrations as postgres user, which creates all the tables in here with postgres as the owner
3. Run 1.8 relinquish, which changes owner of tables created in step 2 to have dockstore as the owner.

Instead we could just run step 2 as the dockstore user, and skip step 3. It also has the advantage of not needing to update two migration files very time somebody adds a table, and creating 2 migration files for every release.

Unless there are permissions isssues, I feel like this is a better approach.

@denis-yuen Does that sound reasonable, or are there some issues I'm overlooking?

Reply:
I think it makes sense. 
I had started a similar thought somewhere else, but didn't work it completely out. 
I think it makes sense that our migrations change users starting with 1.8